tests/cppe/scf-hf/input.dat

```diff
@@ -1,9 +1,19 @@
+#          Polarizability tensor for frequency     0.000000 au
+#          ----------------------------------------------------
+#                            Ex             Ey             Ez
+#
+#                Ex    36.14995      -26.70382       3.599036
+#                Ey   -26.70384       35.10354       8.957926
+#                Ez    3.598892       8.958217       78.45963
+
 #! PE-SCF of PNA in presence of 6 water molecules
-#! Reference data from Q-Chem calculation
+#! Reference data from Dalton
```

Comment:
Was this DALTON or LSDALTON?

Reply:
I've run the PE calculation with Dalton.

Reply:
Shoot I commented in the wrong place! I meant to comment the PCM reference output.